src/zenml/models/v2/base/filter.py

```diff
@@ -598,6 +648,19 @@ def _resolve_operator(value: Any) -> Tuple[Any, GenericFilterOps]:
             ):
                 value = split_value[1]
                 operator = GenericFilterOps(split_value[0])
+
+            if operator == operator.ONEOF:
+                try:
+                    value = json.loads(value)
```

Comment:
Mhm I'm wondering whether json is the best format here? Wouldn't that be very weird as e.g. a query param in the URL? `/stacks?id=oneof:'["<ID>", "ID"]'`?

Reply:
As we discussed, I will leave this as it is. But there is definitely room for improvement here.